programming/sublime-text/pspec.xml

```diff
@@ -20,17 +22,39 @@
     </Source>
 
     <Package>
-        <Name>sublime-text-3</Name>
+        <Name>sublime-text</Name>
+        <Summary xml:lang="en">Sublime Text is a sophisticated text editor for code, markup and prose.</Summary>
+        <Description xml:lang="en"> Sublime Text is a sophisticated text editor for code, markup and prose.</Description>
+        <PartOf>programming</PartOf>
         <Files>
-            <Path fileType="*">/</Path>
+            <Path fileType="data">/opt/sublime_text</Path>
+            <Path fileType="data">/usr/bin/sublime_text</Path>
+            <Path fileType="data">/usr/share/applications/sublime_text.desktop</Path>
         </Files>
+        <AdditionalFiles>
+            <AdditionalFile owner="root" permission="0644" target="/usr/share/applications/sublime_text.desktop">sublime_text.desktop</AdditionalFile>
+        </AdditionalFiles>
         <RuntimeDependencies>
             <Dependency>libxscrnsaver</Dependency>
             <Dependency>gconf</Dependency>
         </RuntimeDependencies>
     </Package>
 
     <History>
+        <Update release="11">
+            <Date>18-07-2021</Date>
+            <Version>4113</Version>
+            <Comment>Update to ST4 Build 4113</Comment>
+            <Name>Abhinav Kulshreshtha</Name>
+            <Email>AbhinavOther@gmail.com</Email>
+        </Update>
+        <Update release="10">
+            <Date>26-05-2021</Date>
+            <Version>4107</Version>
+            <Comment>Update to 4.0.0 Build 4107</Comment>
+            <Name>Abhinav Kulshreshtha</Name>
+            <Email>AbhinavOther@gmail.com</Email>
+        </Update>
```

Comment:
Should probably squash those releases into one for the initial PR.

Also date format is "MM-DD-YYYY" here

Reply:
I will change the date format as you asked. I am in India, we have dd-mm format. I usually use ISO format YYYY-MM-DD when contributing, This must have slipped here because I forked this late at night out of necessity. 

Can you guide me on proper commands for squashing on an open PR?  I have already updated my fork with Build 4113, but you are commenting on 4107, So I guess that update didn't automatically reflected here.

Reply:
Yeah, I'm also always tempted to use DD-MM because that's how it is here in Austria, no worries.

Sorry, I wasn't being clear enough. I see both your updates. I didn't mean you have to necessarily squash the commits (though that would also be neat), but that you have two releases in the pspec.xml (10 and 11), but it would be enough to only have one, that contains all the changes and the update to the newest version, since from the point of view of other Solus users there will just be a single update/release once this is merged. No biggie though. Like so:

```
       <Update release="10">
            <Date>07-18-2021</Date>
            <Version>4113</Version>
            <Comment>Update to ST4 Build 4113</Comment>
            <Name>Abhinav Kulshreshtha</Name>
            <Email>AbhinavOther@gmail.com</Email>
        </Update>
```